bin/create_db.sh

```diff
@@ -19,22 +19,66 @@ psql -c "CREATE DATABASE chronicle ENCODING 'UTF-8' LC_COLLATE = 'en_US.UTF-8' L
 psql -c 'GRANT ALL PRIVILEGES ON DATABASE chronicle to chronicle;' -U $PSQLUSER
 psql -c 'ALTER SCHEMA public OWNER TO chronicle;' -U $PSQLUSER
 psql -c "CREATE TABLE IF NOT EXISTS users (
-  fxaId CHAR(32) PRIMARY KEY,
+  user_id CHAR(32) PRIMARY KEY,
   email VARCHAR(255) NOT NULL,
-  oauthToken TEXT,
-  createdAt TIMESTAMPTZ(3) NOT NULL,
-  updatedAt TIMESTAMPTZ(3)
+  oauth_token TEXT,
+  created_at TIMESTAMPTZ(3) NOT NULL,
+  updated_at TIMESTAMPTZ(3)
 );" -d chronicle -U chronicle
-psql -c "CREATE TABLE IF NOT EXISTS visits (
+psql -c "CREATE TABLE IF NOT EXISTS user_pages (
   id UUID PRIMARY KEY,
-  fxaId CHAR(32) REFERENCES users,
-  url TEXT NOT NULL,
-  rawUrl TEXT NOT NULL,
-  urlHash CHAR(40) NOT NULL,
+  user_id CHAR(32) REFERENCES users(user_id),
+  url VARCHAR(2048) NOT NULL,
+  raw_url VARCHAR(2048) NOT NULL,
+  url_hash CHAR(40) NOT NULL,
   title TEXT NOT NULL,
-  visitedAt TIMESTAMPTZ(3) NOT NULL,
-  updatedAt TIMESTAMPTZ(3)
+  extracted_at TIMESTAMPTZ(3),
+  extracted_author_name TEXT,
+  extracted_author_url VARCHAR(2048),
+  extracted_cache_age BIGINT,
+  extracted_content TEXT,
+  extracted_description TEXT,
+  extracted_favicon_color TEXT,
+  extracted_favicon_url VARCHAR(2048),
+  extracted_image_caption TEXT,
+  extracted_image_color TEXT,
+  extracted_image_entropy DOUBLE PRECISION,
+  extracted_image_height INTEGER,
+  extracted_image_url VARCHAR(2048),
+  extracted_image_width INTEGER,
+  extracted_language TEXT,
+  extracted_lead TEXT,
+  extracted_media_duration INTEGER,
+  extracted_media_height INTEGER,
+  extracted_media_html TEXT,
+  extracted_media_type TEXT,
+  extracted_media_width INTEGER,
+  extracted_provider_display TEXT,
+  extracted_provider_name TEXT,
+  extracted_provider_url VARCHAR(2048),
+  extracted_published TIMESTAMPTZ(3),
+  extracted_safe BOOLEAN,
+  extracted_title TEXT,
+  extracted_type TEXT,
+  extracted_url VARCHAR(2048),
+  created_at TIMESTAMPTZ(3) NOT NULL,
+  updated_at TIMESTAMPTZ(3) NOT NULL
```

Comment:
Good catch! I'll make updated_at mandatory everywhere.

The reason for the change, btw, is that elasticsearch's [optimistic concurrency](http://www.elasticsearch.org/guide/en/elasticsearch/guide/current/optimistic-concurrency-control.html) model relies on queries including an optional version identifier; the queries will bail if the version is wrong. Since postgres is (for now) our primary datastore, we want to use the postgres `updated_at` timestamp as our ordered, sortable version identifier. But that's a todo, it's not implemented yet - either in ES or in our ES queries. So...maybe I'll just ensure updated_at is always there, for now, in all the tables.